frontend/src/core/editor/components/FailedChecks.js

```diff
@@ -93,7 +93,7 @@ export default function FailedChecks(
 }
 
 type MainActionProps = {|
-    source: number | string,
+    source: string,
```

Comment:
This was originally typed just as a number and I added the string type based on the use below to fix an error that came up during the upgrade. Which makes me wonder a) why it was of type number to begin with and b) why no errors were thrown before.

Reply:
There are many things that I tried to break that didn't trigger flow errors. I was pretty underwhelmed by the amount of typesafety in flow.